File: hop-build-payment-form.php
<form action="https://paynow-sandbox.ipsi.com.au/bespoke/latitude" method="post">
    <?php
    while(list($key, $value) = each($output)){
    ?>
      <input type="hidden" name="<?php echo $key ?>" value="<?php echo $value ?>" />
    <?php } ?>
    <input type="submit" value="Checkout with Interest Free" />
</form>

File: hop-build-payment-form.java
url = "https://onlinepaymentintegration.gemoney.com.au/eapps/ApplyRetail.faces";
out.print("<html>\n");
out.print("<form name=\"merchantForm\" action=\"" + url + "\" method=\"POST\">\n");
out.print("<input type=\"hidden\" name=\"merchantID\" value=\"" + output.get(MerchantRequestParameter.MERCHANT_ID) + "\">\n");
out.print("<input type=\"hidden\" name=\"source\" value=\"" + output.get(MerchantRequestParameter.SOURCE_FLAG) + "\">\n");
out.print("<input type=\"hidden\" name=\"merchant_eappsData\" value=\"" + output.get(MerchantRequestParameter.IN_DATA_BLOCK) + "\">\n");
out.print("<input type=\"submit\" name=\"Submit\" value=\"Submit\">\n");
out.print("</form>");
out.print("</html>");

